src/dialogs/more-info/components/lights/dialog-light-color-picker.ts

```diff
@@ -0,0 +1,599 @@
+import "@material/mwc-button";
+import "@material/mwc-tab-bar/mwc-tab-bar";
+import "@material/mwc-tab/mwc-tab";
+import { mdiPalette } from "@mdi/js";
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  PropertyValues,
+  TemplateResult,
+} from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import "../../../../components/ha-control-slider";
+import "../../../../components/ha-button-toggle-group";
+import "../../../../components/ha-color-picker";
+import "../../../../components/ha-dialog";
+import "../../../../components/ha-header-bar";
+import "../../../../components/ha-icon-button-prev";
+import "../../../../components/ha-labeled-slider";
+import {
+  getLightCurrentModeRgbColor,
+  LightColorMode,
+  LightEntity,
+  lightSupportsColorMode,
+} from "../../../../data/light";
+import { haStyleDialog } from "../../../../resources/styles";
+import { HomeAssistant } from "../../../../types";
+import { LightColorPickerDialogParams } from "./show-dialog-light-color-picker";
+
+type Mode = "color_temp" | "color";
+
+@customElement("dialog-light-color-picker")
+class DialogLightColorPicker extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _params?: LightColorPickerDialogParams;
+
+  @state() private _ctSliderValue?: number;
+
+  @state() private _cwSliderValue?: number;
+
+  @state() private _wwSliderValue?: number;
+
+  @state() private _wvSliderValue?: number;
+
+  @state() private _colorBrightnessSliderValue?: number;
+
+  @state() private _brightnessAdjusted?: number;
+
+  @state() private _hueSegments = 24;
+
+  @state() private _saturationSegments = 8;
+
+  @state() private _colorPickerColor?: [number, number, number];
+
+  @state() private _mode?: Mode;
+
+  @state() private _modes: Mode[] = [];
+
+  get stateObj() {
+    return this._params
+      ? (this.hass.states[this._params.entityId] as LightEntity)
+      : undefined;
+  }
+
+  public async showDialog(params: LightColorPickerDialogParams): Promise<void> {
+    this._params = params;
+
+    const supportsTemp = lightSupportsColorMode(
+      this.stateObj!,
+      LightColorMode.COLOR_TEMP
+    );
+
+    const modes: Mode[] = [];
+    modes.push("color");
```

Comment:
The light can support color temp but not color

Reply:
I kept the previous logic
```
  private _toggleButtons = memoizeOne(
    (supportsTemp: boolean, supportsWhite: boolean) => {
      const modes = [{ label: "Color", value: "color" }];
      if (supportsTemp) {
        modes.push({ label: "Temperature", value: LightColorMode.COLOR_TEMP });
      }
      if (supportsWhite) {
        modes.push({ label: "White", value: LightColorMode.WHITE });
      }
      return modes;
    }
  );
  ```

Reply:
I also dropped the white mode (we need to discuss that point in UX meeting)

Reply:
that was because if there where multiple modes (so toggle buttons), one them would always be `color`, but now we show it always, even when there is only 1 mode